Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM ubuntu
+RUN echo 'Building new SuperDuper Docker image...'
+COPY test/testdata/ca-cert.pem /tmp/
+RUN apt-get install -y software-properties-common && \
```

Comment:
Oh, ok, I didn't know that was the package that ```apt-add-repository``` came from... As you were, then.